url.bs

```diff
@@ -164,11 +164,11 @@ contains bytes that are not <a>ASCII bytes</a> might be insecure and is not reco
 
      note that query and application/x-www-form-urlencoded use their own
      local sets -->
-<p>The <dfn>simple encode set</dfn> are <a>C0 controls</a> and all code points greater
-than U+007E.
+<p>The <dfn>C0 control encode set</dfn> are <a>C0 controls</a> and all code points greater than
```

Comment:
It sounds like there's no good unifying name for those concepts, so I guess naming it after C0 controls is fine. *Shrug*.